cogs/utils/paginator.py

```diff
@@ -0,0 +1,268 @@
+import asyncio
+import discord
+from discord.ext.commands import Paginator as CommandPaginator
+from discord.ext.commands import Context
+import typing as t
+
+FIRST_EMOJI = "\u23EE"  # [:track_previous:]
```

Comment:
```suggestion
import typing as t


FIRST_EMOJI = "\u23EE"  # [:track_previous:]
```
Flake8 enforces 2 blank lines after imports